pkg/cmd/create_job.go

```diff
@@ -0,0 +1,221 @@
+	return &CreateJobOptions{
+		configFlags:       genericclioptions.NewConfigFlags(true),
+		previewPrintFlags: genericclioptions.NewJSONYamlPrintFlags(),
+		printFlags:        genericclioptions.NewPrintFlags("created").WithTypeSetter(scheme.Scheme),
+		IOStreams:         streams,
+	}
+}
+
+// AddFlags adds a flag to the flag set.
+func (o *CreateJobOptions) AddFlags(flags *pflag.FlagSet) {
+	flags.StringVar(&o.from, "from", o.from, "The name of the resource to create a Job from (only cronjob is supported).")
+
+	// original flags
+	flags.BoolVarP(&o.Preview, "preview", "P", false,
+		"If true, display the object YAML|JSON by preview window for fuzzy finder selector.")
+	flags.StringVar(&o.PreviewFormat, "preview-format", "yaml",
+		"Preview window output format. One of json|yaml.")
+	flags.BoolVar(&o.RawPreview, "raw-preview", false,
+		"If true, display the unsimplified object in the preview window. (default is simplified)")
+}
+
+// Complete sets all information required for get logs.
+func (o *CreateJobOptions) Complete(cmd *cobra.Command, args []string) error {
+	client, err := kubernetes.NewClient(o.configFlags)
+	if err != nil {
+		return fmt.Errorf("failed to new Kubernetes client: %w", err)
+	}
+
+	if o.from == "" {
+		return fmt.Errorf("--from option is required, only supported job from cronjob")
+	}
+
+	if o.from != "cronjob" {
+		return fmt.Errorf("must specify resource, only supported job")
+	}
+
+	if len(args) >= 1 {
+		o.name = args[0]
+	}
+
+	o.cronJobClient = client.BatchV1beta1()
+	o.jobClient = client.BatchV1()
+
+	kubeConfig := o.configFlags.ToRawKubeConfigLoader()
+
+	namespace, _, err := kubeConfig.Namespace()
+	if err != nil {
+		return fmt.Errorf("faild to get namespace from kube config: %w", err)
+	}
+
+	o.namespace = namespace
+
+	printer, err := o.printFlags.ToPrinter()
+	if err != nil {
+		return err
+	}
+
+	o.printObj = func(obj runtime.Object) error {
+		return printer.PrintObj(obj, o.Out)
+	}
+
+	return nil
+}
+
+// Validate ensures that all required arguments and flag values are provided.
+func (o *CreateJobOptions) Validate() error {
+	return nil
+}
+
+// Run execute fizzy finder and create job from cronJob.
+func (o *CreateJobOptions) Run(ctx context.Context) error {
+	cronJobs, err := o.cronJobClient.CronJobs(o.namespace).List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return fmt.Errorf("failed to list cronJobs: %w", err)
+	}
+
+	var printer printers.ResourcePrinter
+	if o.Preview {
+		printer, err = o.previewPrintFlags.ToPrinter(o.PreviewFormat)
+		if err != nil {
+			return err
+		}
+	}
+
+	cronJob, err := fuzzyfinder.CronJobs(cronJobs.Items, printer, o.RawPreview)
+	if err != nil {
+		return fmt.Errorf("failed to fuzzyfinder execute: %w", err)
+	}
+
+	job := o.createJobFromCronJob(&cronJob, &o.name)
+
+	createOptions := metav1.CreateOptions{}
+	res, err := o.jobClient.Jobs(cronJob.Namespace).Create(context.Background(), job, createOptions)
```

Comment:
```suggestion
	res, err := o.jobClient.Jobs(cronJob.Namespace).Create(context.Background(), job, metav1.CreateOptions{})
```

I think you can sum it up in one line.